src/Resources/views/Admin/ContactRequest/Crud/show.html.twig

```diff
@@ -33,6 +33,22 @@
             <p>
                 {{ resource.email|default('') }}
             </p>
+            <h2>{{ 'monsieurbiz.contact_request.form.name'|trans }}</h2>
+            <p>
+                {{ resource.name|default('') }}
+            </p>
+            <h2>{{ 'monsieurbiz.contact_request.form.company'|trans }}</h2>
+            <p>
+                {{ resource.company|default('') }}
+            </p>
+            <h2>{{ 'monsieurbiz.contact_request.form.phone_number'|trans }}</h2>
+            <p>
+                {{ resource.phone_number|default('') }}
+            </p>
+            <h2>{{ 'monsieurbiz.contact_request.ui.email'|trans }}</h2>
+            <p>
+                {{ resource.email|default('') }}
+            </p>
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought:** ‏do not display your custom fields if they are empty ?

Reply:
here you go